ci-operator/step-registry/openshift/e2e/ibmcloud/heterogeneous/upi/OWNERS

```diff
@@ -1,10 +1,12 @@
 approvers:
 - Shilpa-Gokul
 - prb112
+- pkenchap
 - werled
 - multiarch-approvers
 reviewers:
 - Shilpa-Gokul
 - prb112
+- pkenchap
 - werled
-- multiarch-reviewers
\ No newline at end of file
+- multiarch-reviewers
```

Comment:
May need to check there is no new-line at the end of this file. Multiarch-reviewers shouldn't be flagged as a change

Reply:
yes , it was added from the vim , I removed using the truncate command .